src/components/content-tab-assistant.tsx

```diff
@@ -483,11 +484,7 @@ export function ContentTabAssistant( { selectedSite }: ContentTabAssistantProps
 						isAssistantThinking={ isAssistantThinking }
 					/>
 					<div data-testid="guidelines-link" className="text-a8c-gray-50 self-end py-2">
-						{ createInterpolateElement( __( 'Powered by experimental AI. <a>Learn more</a>' ), {
-							a: (
-								<Button variant="link" onClick={ () => getIpcApi().openURL( AI_GUIDELINES_URL ) } />
-							),
-						} ) }
+						{ __( 'Powered by experimental AI.' ) } <LearnMoreLink docsLinksKey="a8cAiGuidelines" />
```

Comment:
@nightnei , let's keep the `createInterpolateElement` call. That way it will look better for RTL languages. Something like:

```suggestion
{ createInterpolateElement( __( 'Powered by experimental AI. <learn_more_link />' ), {
	learn_more_link: <LearnMoreLink docsLinksKey="a8cAiGuidelines" />,
} ) }
```

Reply:
Great catch, I will update it 👍